src/celengine/meshmanager.h

```diff
@@ -99,4 +99,6 @@ inline bool operator<(const GeometryInfo::ResourceKey& k0,
 
 using GeometryManager = ResourceManager<GeometryInfo>;
 
-extern GeometryManager* GetGeometryManager();
+inline GeometryManager geometryManager("models");
```

Comment:
only trivially constructable objects can be static. please return the old code.

Reply:
returned.